client/src/components/EventForm/index.jsx

```diff
@@ -93,15 +131,15 @@ class EventForm extends Component {
                 this.setState({ [key]: momentUTCToLocal(data.event[key]).format('D MMMM, YYYY') });
 
                 if (key === 'start_date') {
-                  this.setState({ start_time: momentUTCToLocal(data.event[key]).format('h:mmA') });
+                  this.setState({ start_time: momentUTCToLocal(data.event[key]).format('HH:mm') });
```

Comment:
What time format that HH:mm?

Reply:
[Docs](https://momentjs.com/docs/#/parsing/string-format/)